app/policies/work_policy.rb

```diff
@@ -13,4 +13,15 @@ def limit
   def show?
     record.discover_access?(user) || user.admin?
   end
+
+  def edit?
+    editable?
+  end
+
+  private
+
+    # @todo There's a bug in the permissions because a depositor should have edit access by default
+    def editable?
+      record.edit_access?(user) || record.depositor == user.actor || user.admin?
```

Comment:
Can we move this to ApplicationPolicy? I think it would clear up the confusion. Also, I don't think it's a bug anymore because this was put it back before we decided that depositors would not have their own ACL.

Reply:
We are going to 🏈 on this one, and move it into a new ticket, because it has some bigger implications and refactoring possibilities.

Reply:
@rschenk Let's pass on this. It's going to take more work that originally though, but I'll ticket it for future reference. See #586